#!/bin/sh /usr/share/dpatch/dpatch-run
## 04_solaris_spcific_fixes.dpatch by  <kartik.mistry@gmail.com>
## Thanks to Mario Lang for this!
## DP: Fixes of solaris specific in src/orca/orca (Closes: #408875)

@DPATCH@

--- gnome-orca-2.17.5-orig/src/orca/orca	2007-02-02 14:15:17.000000000 +0530
+++ gnome-orca-2.17.5/src/orca/orca	2007-02-02 14:17:49.000000000 +0530
@@ -28,9 +28,12 @@
 # __copyright__ = "Copyright (c) 2005-2006 Sun Microsystems Inc."
 # __license__   = "LGPL"
 
+# We are making set -e required!
+set -e
+
 # Set the user's $PATH for this script.
-#
-export PATH="/usr/ccs/bin:/usr/bin:/usr/sbin:/bin:/usr/sfw/bin:/usr/openwin/bin:/usr/X11R6/bin"
+# Removing Solaris specific paths from below
+export PATH="/usr/bin:/usr/sbin:/bin:/usr/X11R6/bin"
 
 # If you set RUNONCE to "true", then this will just run Orca once and quit.
 #

